myapp/deleteClient.js

```diff
@@ -0,0 +1,21 @@
+const request = require('request');
+const app = require('./app.js');
+
+deleteClient = function () { 
+  app.delete('/clients/:client', function (req, res) {
+    request({
+      url: 'https://api.createsend.com/api/v3.1/clients/'+req.params.client+'.json',
+      method: "DELETE",
+      },
+      function (err, response, data) {
+        if(err) {
+          res.send(err);
+          return(err);
```

Comment:
Line 12-13 can be combined into one,
`return res.send(err);`
This will serve 2 purposes
- return breaks the further execution in case of error
- Error is sent in the response.